src/BigDecimal.php

```diff
@@ -770,10 +770,12 @@ public function unserialize($value) : void
     /**
      * Puts the internal values of the given decimal numbers on the same scale.
      *
-     * @param BigDecimal $x The first decimal number.
-     * @param BigDecimal $y The second decimal number.
-     * @param string     $a A variable to store the scaled integer value of $x.
-     * @param string     $b A variable to store the scaled integer value of $y.
+     * @param BigDecimal  $x The first decimal number.
+     * @param BigDecimal  $y The second decimal number.
+     * @param string|null $a A variable to store the scaled integer value of $x.
```

Comment:
Why is this `string|null`? I understand that when the method is called without `$a` being set, it's `null` as a result, but I really feel like it makes the documentation misleading. Is `@param-out` acceptable without `@param` at all?

Reply:
Reverted `null` changes. With `@param-out` is works just fine.